src/test/regress/expected/failure_split_cleanup.out

```diff
@@ -95,7 +95,7 @@ ERROR:  connection to the remote node localhost:xxxxx failed with the following
 
     \c - postgres - :master_port
     CALL pg_catalog.citus_cleanup_orphaned_resources();
-NOTICE:  cleaned up 79 orphaned resources
+NOTICE:  cleaned up 49 orphaned resources
```

Comment:
This is a flaky test: https://github.com/citusdata/citus/pull/6450 I added you as a reviewer for this test I think we should merge that PR first.